src/main/java/org/alfresco/rest/api/impl/GroupsImpl.java

```diff
@@ -213,7 +221,7 @@ public Group getGroup(String groupId, Parameters parameters) throws EntityNotFou
         PagingResults<AuthorityInfo> pagingResult;
         try
         {
-            pagingResult = getAuthoritiesInfo(authorityType, isRootParam, zoneFilter, rootAuthorities, sortProp, paging);
+            pagingResult = getAuthoritiesInfo(authorityType, isRootParam, zoneFilter, displayNameFilter, rootAuthorities, sortProp, paging);

```

Comment:
This method has too many parameters, please refactoring it, at least using a parameter object.